### ISSUE TITLE
 README.md file seems to be outdated

### ISSUE THREAD

**Issue by LuckyFellow** [nZEDb/nZEDb#2555]
This README.md file seems to be outdated:  nZEDb/www/themes/README.md

**Comment by niel** [comment]
I committed some quick changes to the dev branch https://github.com/nZEDb/nZEDb/commit/c019507a8bfdad309178d34ede8b4e66b7f95c99 . If they're not what you meant, can you be more specific about what is no longer accurate, please.
Also, you could make a Pull Request with the changes.

**Comment by LuckyFellow** [comment]
Thank you very much.
That is exactly what I noticed.

Also I struggle(d) with the additional instructions:

```
Change <link href="{$smarty.const.WWW_TOP}/themes/Simple/styles/style.css" rel="stylesheet" media="screen">

To <link href="{$smarty.const.WWW_TOP}/themes/mytheme/styles/style.css" rel="stylesheet" media="screen">
```

I didn't copy the 'simple' theme as in the example, but the 'Alpha' theme. Though I guess it makes no big of a difference. The themes are interwined with the 'Default' theme as well as the shared folder.
I can not rename the 'Default' folder so that it gets hidden, because there are references to the 'Default' theme at least in:
www\pages\AdminPage.php
www\pages\Page.php

If I change those php files, it becomes more and more tricky to update later.
It would be awesome if there was a solution to this explained in the README.md file.
If there is no solution to this situation, it would be nice if this was addressed in the README.md file.
